PyInstaller/loader/pyimod02_archive.py

```diff
@@ -147,10 +147,7 @@ def __init__(self, path=None, start=0):
             # We cannot use at this bootstrap stage importlib directly
             # but its frozen variant.
             import _frozen_importlib
-            if sys.version_info[1] <= 3:
-                # Python 3.3
-                self.pymagic = _frozen_importlib._MAGIC_BYTES
-            elif sys.version_info[1] == 4:
+            if sys.version_info[:2] == (3, 4):
```

Comment:
Should this run on Python 4.4, or only Python 3.4?

https://astrofrog.github.io/blog/2016/01/12/stop-writing-python-4-incompatible-code/

Reply:
Since Python 3.4 is EOL 2019-03-16, and not even Python 4.0 is even planned. Thus I'm convinced this code will be dropped much earlier then 4.4 will appear :-)